src/screens/WalletConnectApprovalSheet.tsx

```diff
@@ -211,11 +211,19 @@ export function WalletConnectApprovalSheet() {
    * v2.
    */
   const approvalNetworkInfo = useMemo(() => {
-    const chain = useBackendNetworksStore.getState().getDefaultChains()[approvalChainId || ChainId.mainnet];
+    const chain = useBackendNetworksStore.getState().getDefaultChains()[approvalChainId];
+    if (!chain) {
+      return {
+        chainId: approvalChainId,
+        color: colors.alpha(colors.blueGreyDark, 0.3),
+        name: useBackendNetworksStore.getState().getChainsLabel()[approvalChainId],
```

Comment:
If the chain is falsy won't the name be undefined here? Not sure what the alternative is other than displaying "unknown network" or throwing an error.